scripts/postprocess-docs.py

```diff
@@ -0,0 +1,47 @@
+import json, os, re, fileinput, getopt, sys, urllib.request
+
+base_url=''
+docs_root=''
+
+try:
+  opts, args = getopt.getopt(sys.argv[1:], "b:d:")
+except getopt.GetoptError:
+  print('post-document.py -b <base_url> -d <docs_root>')
+  sys.exit(2)
+for opt, arg in opts:
+  if opt in ("-b", "--baseurl"):
+     base_url = arg
+  elif opt in ("-d", "--docsroot"):
+     docs_root = arg
+     
+# receive, parse and remove index file
+index_file = "search.json"
+urllib.request.urlretrieve('{base_url}/search.json'.format(base_url=base_url), index_file)
```

Comment:
There should be a way to load data in memory and use `json.loads`

Reply:
Right, that's a bit simpler. Updated.